package.json

```diff
@@ -27,7 +27,6 @@
   },
   "dependencies": {
     "adbkit": "^2.4.1",
-    "appium-adb": "^6.0.1",
```

Comment:
Why is this being removed? `appium-adb` is still used in this package. Unfortunately linting doesn't complain this because of npm 3's flat dependency structure.

Reply:
That was a mistake.